app/renderer/js/main.js

```diff
@@ -159,6 +160,35 @@ class ServerManagerView {
 		this.toggleSidebar(showSidebar);
 	}
 
+	async queueDomain(domain) {
+		try {
+			const serverConf = await DomainUtil.checkDomain(domain, {override: true});
+			await DomainUtil.addDomain(serverConf);
+			return true;
+		} catch (err) {
+			logger.error(err);
+			logger.error('Could not add ' + domain + '. Please contact your system administrator.');
```

Comment:
Hmm, do you mean to put this in dialog? Just checking since this is going to log file and this seems more like user-facing error message.

Reply:
No, I figured that would be annoying, because any domains not added yet and present in the config file will be checked and added, so any incorrect domains will keep opening up a dialog every time the app loads. 
We could probably show a dialog the first time that this fails and then only log the messages from then on. What do you think?

Reply:
@priyank-p I've added a commit that shows an error dialog when none of the organizations in `presetOrgs` can be added. If you think we need to be more detailed, then let's use this dialog to list all domains that failed to load. 
(If we just replace the `logger` statement with the `dialog`, then every failing org will trigger an error dialog).